cms/templates/widgets/header.html

```diff
@@ -67,9 +67,11 @@ <h3 class="title"><span class="label"><span class="label-prefix sr">${_("Course"
                   </li>
 
                   ### FUN: Add a button to our video upoader
+                  % if hasattr(settings, 'FUN_DEFAULT_VIDEO_PLAYER'):
```

Comment:
If `settings.FUN_DEFAULT_VIDEO_PLAYER` is set to `None` the video uploader link will be rendered. AFAIU this is not what you want. Can you write it as follows in Mako?

```mako
% if getattr(settings, 'FUN_DEFAULT_VIDEO_PLAYER', None) is not None:
```

Reply:
😱